Ingress/echoheaders/nginx.conf

```diff
@@ -0,0 +1,45 @@
+events {
+    worker_connections 1024;
+}
+
+http {
+    default_type 'text/plain';
+    client_max_body_size 10m;
+
+    server {
+        listen 8080 default_server reuseport;
```

Comment:
can you add oneline comments above nginx config terms (like reuseport, client_max_body_size etc)? Mostly just describe why you're setting it/what happens when we exceed it or it isn't set.